VideoAPI/VideoApi.DAL/Commands/CreateRoomCommand.cs

```diff
@@ -45,7 +44,7 @@ public async Task Handle(CreateRoomCommand command)
 
             var room = new Room(command.ConferenceId, command.Label, command.Type);
 
-            _context.Rooms.Add(room);
+            await _context.Rooms.AddAsync(room);
```

Comment:
Doesnt need to be async, although its doing any harm, maybe just for the future
https://docs.microsoft.com/en-us/dotnet/api/microsoft.entityframeworkcore.dbcontext.addasync?view=efcore-5.0

_This method is async only to allow special value generators, such as the one used by 'Microsoft.EntityFrameworkCore.Metadata.SqlServerValueGenerationStrategy.SequenceHiLo', to access the database asynchronously. For all other cases the non async method should be used._

Reply:
reverted back to sync method